distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -484,4 +480,17 @@ private void setClientAuthConfiguration(String clientAuth) throws Exception {
                     });
         }
     }
+
+    private void verifyResponse(final Response response, int expectedResults) {
+        // verify FTP PUT resulted in ingest, catalogued data
+        expect("Failed to verify FTP ingest results").within(VERIFY_INGEST_TIMEOUT_SEC,
+                TimeUnit.SECONDS)
+                .until(() -> {
+                    int numOfResults = response.xmlPath()
+                            .getList("metacards.metacard")
```

Comment:
We should also verify that the metacards have the right data in them, e.g., title.